packages/manager-components/src/components/ManagerButton/ManagerButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { PropsWithChildren } from 'react';
+import {
+  OsdsButton,
+  OsdsTooltip,
+  OsdsTooltipContent,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import './translations';
+
+import { useIsAuthorized } from '../../hooks/iam';
+
+export type ManagerButtonProps = PropsWithChildren<{
+  action?: string;
```

Comment:
**nitpick:** Considering if we implement any code-annotations, it would be nice to rename it to `iamAction` to easily identify the actions only related to `iam`.

Reply:
👍